src/main/java/com/training/assignments/Assignment1.java

```diff
@@ -0,0 +1,32 @@
+package com.training.assignments;
+
+public class Assignment1 {
+    public double intToDouble(int value) {
```

Comment:
Please follow the below method naming convention when your method is returning something:
- an number/string -> `get` prefix
- an boolean -> `is` prefix